server/kv.go

```diff
@@ -500,6 +500,13 @@ func (store *store) LoadInstanceFullKey(fullkey string) (Instance, error) {
 		if err := json.Unmarshal(data, &ci); err != nil {
 			return nil, errors.WithMessage(err, fmt.Sprintf("failed to unmarshal stored instance %s", fullkey))
 		}
+		if ci.JWTInstance != nil {
```

Comment:
Do we also need to check for `RawAtlassianSecurityContext != ""`? I think this case will never occur but to be on the safer side.

Reply:
@raghavaggarwal2308 I don't think so. And if it is the case that it isn't defined, we should either return an error here or set `ci.JWTInstance` to `nil`. Otherwise we will run into the nil pointer elsewhere. I think we can assume that is always defined if `ci.JWTInstance` is defined